src/main/kotlin/com/cognifide/gradle/aem/common/instance/Instance.kt

```diff
@@ -36,7 +36,11 @@ open class Instance(protected val aem: AemExtension) : Serializable {
 
     val password = common.obj.string {}
 
-    var bearerToken = common.obj.string {}
+    val bearerToken = common.obj.string {}
```

Comment:
this field is no longer needed as serviceCredentials should be exchanged into token in InstanceHttpClient